### PR TITLE
[All hosts](NAA) update GA date for monthly enterprise channel

### PR DESCRIPTION
**Pull request by davidchesnut**
NAA is now GA for monthly enterprise channel.

### CODE REVIEW

File: docs/requirement-sets/common/nested-app-auth-requirement-sets.md [hunk 1]
```diff
@@ -28,12 +28,12 @@ NAA supports both Microsoft Accounts and Microsoft Entra ID (work/school) identi
...
 | PowerPoint  | In preview | In preview                                           | In preview | In preview on iPad | Not applicable |
 | Word        | In preview | In preview                                           | In preview | In preview on iPad | Not applicable |
 
 > [!IMPORTANT]
-> To use NAA on platforms that are still in preview, join the Microsoft 365 Insider Program (https://insider.microsoft365.com/join) and choose **Current Channel (Preview)**. Don't use NAA in production add-ins for any preview platforms. We invite you to try out NAA in test or development environments and welcome feedback on your experience through GitHub (see the **Feedback** section at the end of this page).
+> To use NAA on platforms that are still in preview, (Word, Excel, and PowerPoint), join the Microsoft 365 Insider Program (https://insider.microsoft365.com/join) and choose **Current Channel (Preview)**. Don't use NAA in production add-ins for any preview platforms. We invite you to try out NAA in test or development environments and welcome feedback on your experience through GitHub (see the **Feedback** section at the end of this page).
```
Comment (samantharamon) [inline docs/requirement-sets/common/nested-app-auth-requirement-sets.md:36]:
```suggestion
> To use NAA on platforms that are still in preview (Word, Excel, and PowerPoint), join the Microsoft 365 Insider Program (https://insider.microsoft365.com/join) and choose **Current Channel (Preview)**. Don't use NAA in production add-ins for any preview platforms. We invite you to try out NAA in test or development environments and welcome feedback on your experience through GitHub (see the **Feedback** section at the end of this page).
```

File: docs/requirement-sets/common/nested-app-auth-requirement-sets.md [hunk 2]
```diff
@@ -28,12 +28,12 @@ NAA supports both Microsoft Accounts and Microsoft Entra ID (work/school) identi
 | Application | Web        | Windows                                              | Mac        | iOS/iPad           | Android        |
 |-------------|------------|------------------------------------------------------|------------|--------------------|----------------|
 | Excel       | In preview | In preview                                           | In preview | In preview on iPad | Not applicable |
-| Outlook     | GA         | GA in Current Channel, Preview in all other channels | GA         | GA (iOS)           | GA             |
+| Outlook     | GA         | GA in Current Channel and Monthly Enterprise Channel, Preview in Semi-Annual Channels | GA         | GA (iOS)           | GA             |
```
Comment (samantharamon) [inline docs/requirement-sets/common/nested-app-auth-requirement-sets.md:31]: Consider updating the ms.date attribute of this article.

**Review by samantharamon — APPROVED** [review]
Added minor formatting comments, but looks good otherwise.

File: docs/requirement-sets/common/nested-app-auth-requirement-sets.md (hunk 2 above)
Comment (davidchesnut) [reply docs/requirement-sets/common/nested-app-auth-requirement-sets.md:31]: done

**Pull request merged**